eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -38,7 +38,10 @@ param(
 
   [Parameter(Mandatory = $true)]
   $PRTitle,
-  $PRBody = $PRTitle
+  $PRBody = $PRTitle,
+
+  [Parameter(Mandatory = $false)]
+  $PRLabel
```

Comment:
This should be `PRLabels`, and while the other parameters don't have a type (they probably should) we should use a type here so we know if this should be a string or an array.